shared/teams/common/selection-popup.tsx

```diff
@@ -262,7 +262,42 @@ const TeamChannelsActions = (_: TeamActionsProps) => {
   )
 }
 
-const ChannelMembersActions = (_: ChannelActionsProps) => null
+const ChannelMembersActions = ({conversationIDKey, teamID}: ChannelActionsProps) => {
+  const dispatch = Container.useDispatch()
+  const members = Container.useSelector(s => s.teams.channelSelectedMembers.get(conversationIDKey))
+  if (!members) {
+    // we shouldn't be rendered
+    return null
+  }
+
+  // Members tab functions
+  const onAddToChannel = () =>
+    dispatch(
+      RouteTreeGen.createNavigateAppend({
+        path: [{props: {teamID, usernames: [...members]}, selected: 'teamAddToChannels'}],
+      })
+    )
+  const onEditRoles = () => console.log('onEditRoles not implemented') // TODO
+  const onRemoveFromChannel = () => console.log('onRemoveFromChannel not implemented') // TODO
+
+  return (
+    <ActionsWrapper>
+      <Kb.Button
+        label="Add to channels"
+        mode="Secondary"
+        onClick={onAddToChannel}
+        fullWidth={Styles.isMobile}
+      />
+      <Kb.Button label="Edit role" mode="Secondary" onClick={onEditRoles} fullWidth={Styles.isMobile} />
+      <Kb.Button
```

Comment:
I thought you weren't supposed to be able to remove people from channels?

Reply:
Design added more information in Zeplin.